Sources/ProjectDescription/PluginLocation.swift

```diff
@@ -33,8 +33,8 @@ public struct PluginLocation: Codable, Equatable {
     /// ```
     /// .git(url: "https://git/plugin.git", sha: "d06b4b3d")
     /// ```
-    public static func git(url: String, sha: String, directory: String? = nil) -> Self {
-        PluginLocation(type: .gitWithSha(url: url, sha: sha, directory: directory))
+    public static func git(url: String, sha: String, directory: String? = nil, releaseUrl: String? = nil) -> Self {
+        PluginLocation(type: .gitWithSha(url: url, sha: sha, directory: directory, releaseUrl: releaseUrl))
     }
```

Comment:
We don't support binaries for sha configured plugins, so you can remove the parameter from here